presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleAssert.java

```diff
@@ -231,12 +230,12 @@ public RuleApplication(Lookup lookup, Map<Symbol, Type> types, Optional<PlanNode
 
         private boolean wasRuleApplied()
         {
-            return result.isPresent();
+            return !result.isEmpty();
         }
 
         public PlanNode getResult()
         {
-            return result.orElseThrow(() -> new IllegalStateException("Rule was not applied"));
+            return result.getTransformedPlan().orElseThrow(() -> new IllegalStateException("Rule was not applied"));
```

Comment:
Also update the message to `"Rule did not produce transformed plan"`. And perhaps rename this method to `getTransformedPlan()`